Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -46,6 +46,7 @@ def tests
 
       files = Dir.glob("test/**/*_test.rb")
                  .reject { |p| !OS.mac? && p.start_with?("test/os/mac/") }
+                 .reject { |p| p.start_with?("test/vendor/bundle/") }
```

Comment:
No strong feelings either way, just figured it was better to be specific in case we decided to test censored code for e.g. integrations at some point.